server/cli/src/cli.rs

```diff
@@ -388,38 +388,46 @@ async fn main() -> anyhow::Result<()> {
         Action::BuildStandardReports {} => {
             let connection_manager = get_storage_connection_manager(&settings.database);
             let con = connection_manager.connection()?;
-            let base_reports_dir = "./reports";
+            let base_reports_dir = Path::new("reports");
 
-            let report_names: Vec<String> = fs::read_dir(base_reports_dir)?
+            let report_names: Vec<PathBuf> = fs::read_dir(base_reports_dir)?
                 .filter_map(|r| r.ok())
                 .map(|e| e.path())
                 .filter(|p| p.is_dir())
-                .filter(|name| name != &PathBuf::from("./reports/generated"))
-                .map(|p| p.into_os_string().into_string().unwrap())
+                .filter(|name| name != &Path::new("reports").join("generated"))
+                .map(|p| p)
                 .collect();
 
             let mut reports_data = ReportsData { reports: vec![] };
 
             for name_dir in report_names {
-                let report_versions: Vec<String> = fs::read_dir(&name_dir)?
+                let report_versions: Vec<PathBuf> = fs::read_dir(&name_dir)?
                     .filter_map(|r| r.ok())
                     .map(|e| e.path())
                     .filter(|p| p.is_dir())
-                    .map(|p| p.into_os_string().into_string().unwrap())
+                    .map(|p| p)
                     .collect();
 
-                let (_, name) = name_dir.rsplit_once('/').unwrap();
+                let parent = name_dir
```

Comment:
what is the reason for this parent and the `name` ? Is it just for logs ?

Reply:
And also the name of the name.json generated output file for that report. 
FYI this is not the name used on the front end as this is extracted from the manifest file.